wrappers/hnyecho/echo.go

```diff
@@ -0,0 +1,81 @@
+package hnyecho
+
+import (
+	"github.com/jamietsao/beeline-go/wrappers/common"
```

Comment:
If/when this gets merged, do I need to change the import path to be `honeycombio` instead of `jamietsao`?

Reply:
Yeah - in fact you can probably move the `github.com/jamietsao/beeline-go` fork to your local `$GOPATH/go/src/github.com/honeycombio/beeline-go` and go ahead and do it now